langlib/lang.array/src/main/ballerina/array.bal

```diff
@@ -82,6 +102,14 @@ public isolated function 'map(Type[] arr, @isolatedParam function(Type val) retu
 #
 # The parameter `func` is applied to each member of parameter `arr` in order.
 #
+# ```
+# string str = "0";
+# [1, 2, 3].forEach(function (int i) {
+#     str += string ` ${i}`;
+# });
+# str ⇒ 0 1 2 3
```

Comment:
```suggestion
# str ⇒ "0 1 2 3"
```
wdyt?

Reply:
No, the idea is to minimally represent what is done by the given langlib function. This require users to understand string interpolation as well. Maybe better to even remove the string concat and just + the ints.
eg:
`sum = sum + i`